support-frontend/assets/components/otherAmount/otherAmount.tsx

```diff
@@ -14,6 +16,8 @@ export type OtherAmountProps = {
 	currency: IsoCurrency;
 	minAmount: number;
 	onOtherAmountChange: (newAmount: string) => void;
+	onBlur?: FocusEventHandler<HTMLInputElement>;
+	onInvalid?: FormEventHandler<HTMLInputElement>;
```

Comment:
given this element is actually just an HTMLInputElement - you could use an [intersection](https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types) of those objects

```ts
type OtherAmountProps = {/*...*/} & HTMLAttributes<HTMLButtonElement>
```

Reply:
That's clever, like that, presume it works because `HTMLInputElement` props are subset of `HTMLAttributes` props. Thanks, changed.